apps/condo/domains/meter/schema/MeterReadingExportTask.js

```diff
@@ -0,0 +1,250 @@
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { extractReqLocale } = require('@open-condo/locales/extractReqLocale')
+
+const { WRONG_VALUE } = require('@condo/domains/common/constants/errors')
+const { COMPLETED, ERROR } = require('@condo/domains/common/constants/export')
+const { EXPORT_STATUS_VALUES, PROCESSING, EXCEL } = require('@condo/domains/common/constants/export')
+const { findAllByKey } = require('@condo/domains/common/utils/ecmascript.utils')
+const { normalizeTimeZone } = require('@condo/domains/common/utils/timezone')
+const access = require('@condo/domains/meter/access/MeterReadingExportTask')
+const { EXPORT_METER_READINGS_MONTHS_LIMIT } = require('@condo/domains/meter/constants/constants')
+const { exportMeterReadings } = require('@condo/domains/meter/tasks/exportMeterReadings')
+const { DEFAULT_ORGANIZATION_TIMEZONE } = require('@condo/domains/organization/constants/common')
+
+
+const { getFileMetaAfterChange } = FileAdapter
+
+const ERRORS = {
+    STATUS_IS_ALREADY_COMPLETED: {
+        code: BAD_USER_INPUT,
+        type: WRONG_VALUE,
+        message: 'Status is already completed',
+        messageForUser: 'api.meterReadingExportTask.STATUS_IS_ALREADY_COMPLETED',
+        mutation: 'updateMeterReadingExportTask',
+        variable: ['data', 'status'],
+    },
+    STATUS_IS_ALREADY_ERROR: {
+        code: BAD_USER_INPUT,
+        type: WRONG_VALUE,
+        message: 'Status is already error',
+        messageForUser: 'api.meterReadingExportTask.STATUS_IS_ALREADY_ERROR',
+        mutation: 'updateMeterReadingExportTask',
+        variable: ['data', 'status'],
+    },
+}
+
+const METER_READING_EXPORT_TASK_FOLDER_NAME = 'MeterReadingExportTask'
+const MeterReadingExportTaskFileAdapter = new FileAdapter(METER_READING_EXPORT_TASK_FOLDER_NAME)
+const setFileMetaAfterChange = getFileMetaAfterChange(MeterReadingExportTaskFileAdapter, 'file')
+
+
+const MeterReadingExportTask = new GQLListSchema('MeterReadingExportTask', {
+    schemaDoc: 'Stores requested export format, status of export job, link to exported file and information about progress of export job',
+    fields: {
+        status: {
+            schemaDoc: 'Status of the job. Used at client side progress information component',
+            type: 'Select',
+            options: EXPORT_STATUS_VALUES,
+            defaultValue: PROCESSING,
+            isRequired: true,
+            access: {
+                create: canOnlyServerSideWithoutUserRequest,
+                read: true,
+                update: true,
+            },
+        },
+
+        format: {
+            schemaDoc: 'Requested export file format',
+            type: 'Select',
+            options: [EXCEL],
+            isRequired: true,
+            access: {
+                create: true,
+                read: true,
+                update: canOnlyServerSideWithoutUserRequest,
```

Comment:
```suggestion
                update: false,
```
Should there be a way to update a format?
I can't think of any such cases